gulp/DevelopmentTasks.ts

```diff
@@ -158,7 +165,15 @@ const doc: ITaskFunction = async () => {
                 continue;
             }
             totalCommands++;
-            const childAnchorTag = "command-" + child.name.replace(/\s/g, "-");
+
+            // create anchor tag that matches full command
+            var childAnchorTag = ""
+            if (oldCommandName !== "") {
+                childAnchorTag = oldCommandName + "-" + definition.name +"-" + child.name.replace(/\s/g, "-");
+            } else {
+                childAnchorTag = definition.name + "-" + child.name.replace(/\s/g, "-");
```

Comment:
You could use template literals here too but not necessary

Reply:
Changed to template literals.